modules/bloop-rifle/src/main/scala/scala/build/blooprifle/VersionUtil.scala

```diff
@@ -1,10 +1,27 @@
 package scala.build.blooprifle
 
+
 object VersionUtil {
-  def jvmRelease(jvmVersion: String): Option[Int] = {
-    val jvmReleaseRegex = "(1[.])?(\\d+)".r
-    jvmReleaseRegex.findFirstMatchIn(jvmVersion).map(_.group(2)).map(_.toInt)
-  }
+
+  /** @param jvmVersion,
+    *   for example '1.8.0.22' or '11.0.2'
+    * @return
+    *   jvm release version (8, 11)
+    */
+  val jvmReleaseRegex = "(1[.])?(\\d+)"
+  def jvmRelease(jvmVersion: String): Option[Int] =
+    jvmReleaseRegex.r.findFirstMatchIn(jvmVersion).map(_.group(2)).map(_.toInt)
```

Comment:
```suggestion
    jvmReleaseRegex.r.findFirstMatchIn(jvmVersion).flatMap(v => Option(v.group(2))).map(_.toInt)
```

Reply:
`null` was impossible there, as the group was required by regex, but I agree we can explicitly handle this case